app/src/main/java/fr/gaulupeau/apps/Poche/network/Updater.java

```diff
@@ -215,7 +223,7 @@ private long performUpdate(ArticlesChangedEvent event, boolean full,
                     article = new Article(null);
                     article.setArticleId(id);
                     article.setTitle(apiArticle.title);
-                    article.setContent(apiArticle.content);
+                    article.setArticleContent(new ArticleContent(null, apiArticle.content));
```

Comment:
Shouldn't we put the id from L226 as id parameter here instead of null?

Reply:
Nope, this is local (SQLite) article ID, which is only available after the `article` is inserted.